kits/default/package.json

```diff
@@ -13,7 +13,7 @@
     "production": "npm-run-all build:*"
   },
   "devDependencies": {
-    "@openeuropa/bcl-builder": "0.14.0",
+    "@openeuropa/bcl-builder": "0.15.0",
```

Comment:
Do not upgrade to this version, is still on development and blocks this if we have to wait until v15 is out.